report/mapping_functions.go

```diff
@@ -48,6 +48,37 @@ func ProcessPID(_ string, m NodeMetadata, grouped bool) (MappedNode, bool) {
 	}, show
 }
 
+// ProcessContainer maps Process topology nodes to the containers they run in.
+// We consider container and image IDs and names to be globally-scoped, and so
+// don't scope them further by e.g. host. If no container metadata is found,
+// nodes are grouped into the Uncontained node. If grouped is true, nodes are
+// grouped by container image.
```

Comment:
Ah OK, so — container names are only used in labels, never as IDs, so indeed we just need to make sure they're not ambiguous there. Will update...